src/elements/inputTags/InputTags.types.ts

```diff
@@ -0,0 +1,35 @@
+interface Label {
+  label: string;
+  field?: string;
+  showOptionalLabel?: boolean;
+  className?: string;
+  style?: object;
+  help?: string;
+  fontWeight?: number;
+  isPrivate?: boolean;
+  privateText?: string;
+  size?: string;
+}
+
+interface Props {
+  tags: Array<string>;
+  name?: string;
+  id?: string;
+  className?: string;
+  dataTestId?: string;
+  placeholder?: string;
+  helperText?: string;
+  disabled?: boolean;
+  seprators?: Array<string>;
+  maxTags?: number;
+  minLength?: number;
+  maxLength?: number;
+  size?: 'sm' | 'md' | 'lg';
+  inputLabelProps?: Label;
```

Comment:
This "inputLabelProps" should have its type imported from the interface of the "InputLabel" component. Can you create a file name "InputLabel.types.ts" with its interface in the component folder, and import that interface in every inputLabelProps prop across the toolkit?

Reply:
Done 👍